doc/modules/biclustering.rst

```diff
@@ -48,6 +48,8 @@ columns:
    :target: ../auto_examples/bicluster/images/sphx_glr_plot_spectral_coclustering_003.png
    :align: center
    :scale: 50
+   :alt: The graph is a square heat map, 5x5, with axes from 0 to 250.  The darkest 5 
+   squares of heat map run diagonally from top left to bottom right.
```

Comment:
Since it hasn't been said on this PR yet, I'm not someone that uses scikit-learn or even knows most of the figures we wrote alt text for at this sprint, so I'm happy to defer to your expertise on how much information is helpful to properly describe the figure in context.

As someone who does know alt text, though, I know two of the things I check alt text for is that is 1. that it describes image in a way that makes sense in the context it's used in 2. that it is a short as possible. I don't think this edit adds much to the meaning of the figure in the documentation since it seems to add details rather than the main features of the figure. Of course, if I'm misunderstanding please let me know and I'll commit this suggestion.

A good explanation of what I'm talking about can be found on [WebAIM's discussion of alt text written for proper context](https://webaim.org/techniques/alttext/#context).